### PR TITLE
Convert System.Xml.XDocument.Streaming.Tests to use Xunit

### PR DESCRIPTION
**Pull request by Maxwe11**
Progress against #642

I recommend to review without whitespace changes
https://github.com/dotnet/corefx/pull/3013/files?w=1

/cc: @krwq

### CODE REVIEW

**Comment by Maxwe11** [comment]
I recommend to review ignoring whitespace changes
https://github.com/dotnet/corefx/pull/3013/files?w=1

**Comment by Maxwe11** [comment]
@dotnet-bot test this please

**Comment by Maxwe11** [comment]
@dotnet-bot test this please

File: src/System.Xml.XDocument/tests/Streaming/StreamingOutput.cs [hunk 1]
```diff
@@ -3,617 +3,571 @@
...
 using System.Xml.Linq;
+
 using System.Xml.XmlDiff;
-using Microsoft.Test.ModuleCore;
+using CoreXml.Test.XLinq;
 
-namespace CoreXml.Test.XLinq
+using Xunit;
+
+namespace XDocumentTests.Streaming
```
Comment (Maxwe11) [inline src/System.Xml.XDocument/tests/Streaming/StreamingOutput.cs]: @Clockwork-Muse what namespace name you could suggest?

Comment (Clockwork-Muse) [inline src/System.Xml.XDocument/tests/Streaming/StreamingOutput.cs]: @Maxwe11 - It's supposed to be ${namespaceOfTypeUnderTest}.Tests, so probably `System.Xml.Linq.Tests`?

**Comment by stephentoub** [comment]
Sorry for the delay, @Maxwe11. Thanks for doing this!  It looks good to me, and since this doesn't introduce the namespace issue and makes other stuff better, I'm going to go ahead and merge it and the namespace issue can be addressed in a subsequent change.

**Pull request merged**